broker/src/main/java/io/camunda/zeebe/broker/partitioning/PartitionManagerImpl.java

```diff
@@ -105,6 +107,12 @@ public ManagedPartitionGroup getPartitionGroup() {
     return partitionGroup;
   }
 
+  public PartitionAdminAccess createAdminAccess(final ConcurrencyControl concurrencyControl) {
+    return new MultiPartitionAdminAccess(
+        concurrencyControl,
+        partitions.stream().map(ZeebePartition::createAdminAccess).collect(Collectors.toList()));
```

Comment:
:upside_down_face: Can we extract this to a temporary variable? I find it much more readable when debugging, since you immediately have the watch. This is really minor though, feel free to ignore.